src/logic/operate.js

```diff
@@ -0,0 +1,24 @@
+import Big from 'big.js';
+
+const operate = (numberOne, numberTwo, operation) => {
+  const display = document.querySelector('.displayOutput');
```

Comment:
- [ ] You shouldn't be using `querySelector`. `operate` and `calculate` functions are merely for logic, and shouldn't get invovled in any DOM manipulation specially when you are using ReactJS. `operate` and `calculate` should only return the results.